src/Common/src/Internal/Runtime/EEType.Constants.cs

```diff
@@ -234,6 +234,8 @@ internal enum CorElementType
 
         ELEMENT_TYPE_ARRAY = 0x14,
 
+        ELEMENT_TYPE_GENERICINST = 0x15,
```

Comment:
Could you fork this enum into `ILCompiler.ReadyToRun` instead of including the file? We would actually like to stop using this enum in ProjectN/CoreRT and delete it (see #478). Adding new references to it will just make the destiny of this less clear (also we should avoid including things with EEType in the name for ready to run for code hygiene reasons).

Reply:
OK, in such a case I should probably copy over the version from CoreFX, S.R.M.Ecma335 because it seems more complete - it's just internal there.